.github/workflows/downstreams.yml

```
@@ -6,9 +6,11 @@ on:
     branches:
       - main
       - 'FEATURE-BRANCH-*'
+  merge_group: 
```

We should be explicit per https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#merge_group

```suggestion
  merge_group:
    types: [checks_requested]
```